### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by Justinezgh**
Hi @dlanzieri I updated the readme file, what do you think? :) There is also a colab link to add a more detailed example!

### CODE REVIEW

**Comment by Justinezgh** [comment]
yup I changed it :) #52

**Comment by Justinezgh** [comment]
yes I think that I already did it in #52  but I may have forgotten one of them

**Comment by EiffL** [comment]
What are we missing on this branch to merge?

File: README.md [hunk 1]
```diff
@@ -1,92 +1,115 @@
...
+  title('Bin %d'%(i+1))
+  axis('off')
 ```
+<p align=center>
+    <img src="img/convergence_map.png" style="width:1000px;">
+</p>
+
+Check out a full example here: [![colab link](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1pSjhrOJbVi80RQlsVz2oXhVAtxwBhSbn?usp=sharing)
 
 # Contributors
```
Comment (dlanzieri) [inline README.md:104]: ## Contributors ✨



<table>
  <tr> 
    <td align="center"><a href="https://aboucaud.github.io"><br /><sub><b>Alexandre Boucaud</b></sub></a><br /><a href="https://github.com/DifferentiableUniverseInitiative/sbi_lens/commits?author=aboucaud" title="Code">💻</a></td>
    <td align="center"><a href="http://flanusse.net"><br /><sub><b>Francois Lanusse</b></sub></a><br /><a href="https://github.com/DifferentiableUniverseInitiative/sbi_lens/commits?author=EiffL" title="Code">💻</a></td>
    <td align="center"><a href="https://www.cosmostat.org/people/denise-lanzieri"><br /><sub><b>Denise Lanzieri</b></sub></a><br /><a href="https://github.com/DifferentiableUniverseInitiative/sbi_lens/commits?author=dlanzieri" title="Code">💻</a></td>
    <td align="center"><a href="https://github.com/Justinezgh"><br /><sub><b>Justine Zeghal</b></sub></a><br /><a href="https://github.com/DifferentiableUniverseInitiative/sbi_lens/commits?author=Justinezgh" title="Code">💻</a></td>
  </tr>
  </tr>
</table>

Comment (dlanzieri) [reply README.md:104]: @Justinezgh You should add a nice picture on Github :-)
And I didn't find your personal web page!

Comment (Justinezgh) [reply README.md:104]: 😬

**Pull request merged**